packages/react/src/elements/components/Flyout.test.js

```diff
@@ -0,0 +1,22 @@
+import React from "react";
```

Comment:
This isn't necessarily related, but I had added this to continue improving the test coverage.